samples/node-js/gce/README.md

```diff
@@ -0,0 +1,81 @@
+# Snapshot Debugger Example for Node.js on Google Compute Engine
+
+NOTE: This sample application was copied from
+[nodejs-getting-started/gce][sample-source]
+and modified for the Snapshot Debugger samples here.
+
+This folder contains the sample code for the [Deploying to Google Compute
+Engine][tutorial-gce]
+tutorial. Please refer to the tutorial for instructions on configuring,
+running, and deploying this sample.
+
+## Enabling Snapshot Debugger
+
+The following code changes have been made to enable the Snapshot Debugger:
+
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/d055bb60c04356ad64695c8853cbfdc59ebe2fdc/samples/node-js/gce/app.js#L17-L24
+
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/d99f0ba821a94203440ba6213378843e172ce214/samples/node-js/gce/package.json#L18-L21
+
+## Create and configure a Compute Engine instance
+
+Note that the Snapshot Debugger requires the following access scopes:
+* https://www.googleapis.com/auth/userinfo.email grants your cluster access to
+your email address.  It is provided below in userinfo-email.
+* https://www.googleapis.com/auth/firebase.database grants your cluster access to
+the Firebase database.
+
+```
+gcloud compute instances create my-app-instance \
+  --image-family=debian-10 \
+  --image-project=debian-cloud \
+  --machine-type=g1-small \
+  --scopes userinfo-email,cloud-platform,https://www.googleapis.com/auth/firebase.database \
+  --metadata-from-file startup-script=startup-script.sh \
+  --zone us-central1-a \
+  --tags http-server
+```
+
+## Determine the Debuggee ID
```

Comment:
Should be a note about waiting for the app to be ready before checking for the debuggee.

Reply:
Added.